spharpy/spherical.py

```diff
@@ -248,6 +248,104 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
+def renormalize(data, channel_convention, current_norm, target_norm, axis=-2):
+    """
+    Renormalize either an array of SH coefficients or SH basis functions.
```

Comment:
```suggestion
    Renormalize SH coefficients or basis functions.
```

Reply:
Since it's not mentioned anywhere. Please spell out SH everywhere for improved readability